src/markdown-pages/docs/ecosystem/ecosystem-architecture.md

```diff
@@ -3,14 +3,14 @@ path: "/docs/ecosystem/architecture"
 title: "Ecosystem Architecture"
 ---
 
-The Galasa ecosystem contains all of the servers and monitors that are required to run Galasa tests in an automated environment or as part of a DevOps pipeline. The following diagram provides a high-level representation of the Galasa ecosystem architecture:<br><br> ![Galasa ecosystem architecture:](ecosystem-overview-final.svg)
+The Galasa ecosystem contains all of the servers and monitors that are required to run Galasa tests in an automated environment or as part of a DevOps pipeline. The following diagram provides a high-level representation of the Galasa ecosystem architecture:<br><br> ![Galasa ecosystem architecture:](ecosystem-overview.svg)
```

Comment:
lets leave the sentence at in an automated environment.

Reply:
done